### PR TITLE
Adds a namespace flag to port-forward.

### PR DESCRIPTION
**Pull request by jdoliner**

### CODE REVIEW

File: src/server/cmd/pachctl/cmd/cmd.go [hunk 1]
```diff
@@ -345,9 +343,10 @@ kubectl %v port-forward "$pod" %d:8080
 			eg.Go(func() error {
 				fmt.Printf("Forwarding the dash (Pachyderm dashboard) UI port to http://localhost:%v ...\n", uiPort)
 				stdin := strings.NewReader(fmt.Sprintf(`
-pod=$(kubectl %v get pod -l app=dash | awk '{if (NR!=1) { print $1; exit 0 }}')
+pod=$(kubectl %v get pod -l app=dash --output='jsonpath={.items[0].metadata.name}')
 kubectl %v port-forward "$pod" %d:8081
 `, kubeCtlFlags, kubeCtlFlags, uiWebsocketPort))
+				fmt.Println(stdin)
```
Comment (sjezewski) [inline src/server/cmd/pachctl/cmd/cmd.go]: Is this debug output? Or something we want the user to see?

File: src/server/cmd/pachctl/cmd/cmd.go [hunk 2]
```diff
@@ -388,108 +388,6 @@ Currently "pachctl garbage-collect" can only be started when there are no active
 		}),
 	}
 
-	var from, to, namespace string
```
Comment (sjezewski) [inline src/server/cmd/pachctl/cmd/cmd.go]: I doubt you meant to remove all this migration stuff. I think you just need a merge

File: src/server/cmd/pachctl/cmd/cmd.go (hunk 1 above)
Comment (jdoliner) [reply src/server/cmd/pachctl/cmd/cmd.go]: Yup, nice catch.

File: src/server/cmd/pachctl/cmd/cmd.go (hunk 2 above)
Comment (jdoliner) [reply src/server/cmd/pachctl/cmd/cmd.go]: It's actually an old implementation of migration from 1.4 to 1.5. Not really useful to anyone anymore and likely to confuse people.

**Pull request merged**